PeerioTechnologies/peerio-desktop
# fix: small Files UI updates

#### Relevant info and issue/PR links 
 https://app.clubhouse.io/peerio/story/10731/desktop-check-that-recent-fixes-and-changes-on-files-got-included-in-big-files-refactor

#### Testing instructions  
Catching the small bugs that fell through the cracks in the big Files refactor.
* reenable the three-dots menu on fileline/folderline that has been selected
* on an individual file's three-dots menu, when you click share or move, the file should be shown as checked/selected under the dialog that pops up (and selection should clear on dialog close)

OK so, if you bulk select multiple files and then use the three-dots menu on one file to move/share, only that one file will be shown as selected (the others will be deselected). If you cancel the dialog instead of moving/sharing, this file will stay selected. Design is deciding whether we want to restore the previous selection state, but for now I think this can go ahead and they can create a new bug if they want to change it.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
